package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "PrivacyPass",
+  "name": "privacypass",
```

Comment:
what's the reason for changing this?

Reply:
npm was warning me about it violating naming conventions, didn't mean to commit that

Reply:
You can revert that

Reply:
Didn't found any offfenders in https://docs.npmjs.com/package-name-guidelines
could you please revert the change.
CI is not happy.

Reply:
![image](https://user-images.githubusercontent.com/44925963/116957090-96430f00-acb4-11eb-9797-45490ad2e0a1.png)
ok

Reply:
Also need to make crypto a global to make CI pass